renderdoc/driver/vulkan/vk_next_chains.cpp

```diff
@@ -1089,6 +1089,105 @@ size_t GetNextPatchSize(const void *pNext)
       }
 #endif
 
+#if ENABLED(RDOC_FUCHSIA)
+        // Image pipe
+        COPY_STRUCT_CAPTURE_ONLY(VK_STRUCTURE_TYPE_IMAGEPIPE_SURFACE_CREATE_INFO_FUCHSIA,
+                                 VkImagePipeSurfaceCreateInfoFUCHSIA);
```

Comment:
These structs will need to be removed from `#define UNHANDLED_STRUCTS()`, and they also need to be handled in the other functions `UnwrapNextChain()` and `CopyNextChainForPatching()`.